### ISSUE TITLE
SvelteKit from prettier to biome: inconsistencies

### ISSUE THREAD

**Issue by frederikhors** [biomejs/biome#4365]
I'm using prettier (unfortunately) in a SvelteKit project, with this .prettierrc file:

```json
{
	"useTabs": true,
	"singleQuote": true,
	"trailingComma": "none",
	"printWidth": 100,
	"plugins": [
		"prettier-plugin-svelte",
		"prettier-plugin-tailwindcss"
	],
	"overrides": [
		{
			"files": "*.svelte",
			"options": {
				"parser": "svelte"
			}
		}
	]
}
```

I migrated this file to biome 1.9.4 and with the below biome.json everything works but I'm getting an annoying difference: the indentation!

With prettier the file is like this:

```svelte
<script lang="ts">
	import { something } from 'somewhere';
	import { something } from 'somewhere';
	import { something } from 'somewhere';

    
    type Props = {
        // something
    }
</script>
```

with Biome now it is like this:

```svelte
<script lang="ts">
import { something } from 'somewhere';
import { something } from 'somewhere';
import { something } from 'somewhere';


type Props = {
    // something
}
</script>
```

Why?

### Configuration

```JSON
{
	"$schema": "https://biomejs.dev/schemas/1.9.4/schema.json",
	"vcs": { "enabled": false, "clientKind": "git", "useIgnoreFile": true },
	"files": { "ignoreUnknown": false, "ignore": [] },
	"formatter": {
		"enabled": true,
		"lineWidth": 100,
	},
	"linter": { "enabled": false },
	"javascript": {
		"formatter": {
			"quoteStyle": "single",
			"trailingCommas": "none"
		}
	},
	"overrides": [{ "include": ["*.svelte"] }]
}
```

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct

**Comment by ematipico** [comment]
This is a [known limitation](https://biomejs.dev/internals/language-support/#html-super-languages-support)

**Comment by frederikhors** [comment]
Can I ask you why is that?

**Comment by dyc3** [comment]
We currently just process the js sections because our HTML parser isn't ready yet.

**Comment by olafurw** [comment]
> We currently just process the js sections because our HTML parser isn't ready yet.

Is there a place where I can monitor the status of the HTML parsing? I'd love to fully switch over to Biome for a pretty large Svelte project I have.

**Comment by frederikhors** [comment]
@dyc3 is still the same today? Can we subscribe to some issue?

@olafurw did you find a way?

**Comment by siketyan** [comment]
We have just implemented an option for that: #7333
